kin-core/src/main/java/kin/core/KinAccount.java

```diff
@@ -84,6 +84,35 @@ TransactionId sendTransactionSync(@NonNull String publicAddress, @NonNull BigDec
     TransactionId sendTransactionSync(@NonNull String publicAddress, @NonNull BigDecimal amount, @Nullable String memo)
         throws OperationFailedException;
 
+    /**
+     * Checks if the account is "burned", which means that this account is not more active.
```

Comment:
not -> no, add "and no kin can be transferred", like in the send burn transaction method.